docs/rules/aws_security_group_rule_invalid_protocol.md

```diff
@@ -0,0 +1,33 @@
+# aws_security_group_rule_invalid_protocol
```

Comment:
Please add a link to this rule in `README.md.tmpl`. I think this rule is categorized as possible errors.
https://github.com/terraform-linters/tflint-ruleset-aws/blob/45cd608fd65235483faa57caa1da9024feb99f1e/docs/rules/README.md.tmpl